src/cascadia/TerminalControl/KeyChord.cpp

```diff
@@ -38,19 +38,68 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         {
             _vkey = MapVirtualKeyW(scanCode, MAPVK_VSC_TO_VK_EX);
         }
+
+        assert(_vkey || _scanCode);
+    }
+
+    uint64_t KeyChord::Hash() const noexcept
+    {
+        // Two KeyChords are equal if they have the same modifiers and either identical
+        // Vkey or ScanCode, with Vkey being preferred. See KeyChord::Equals().
+        // This forces us to _either_ hash _vkey or _scanCode.
+        //
+        // Additionally the hash value with _vkey==123 and _scanCode==123 must be different.
+        // --> Mark the hash of a KeyChord without _vkey with the highest bit set,
```

Comment:
technically not the highest bit, right? it's just one of the bits in the middle-ish

Reply:
I've changed the mixin to `0xBABE0000` and changed the description.